File: single-language-analysis/dataset-files/python/test_smell_133.py
# Original URL: https://github.com/RonJeffries/wordle/blob/e999dceebbe2ee28f5c4b0c3edb675e8b8830d23/test_speed.py#L74-L81

class TestSpeed:

    @pytest.mark.skip("5 seconds is also too long")
    def test_smell_133(self):
        t0 = time.time()
        results = self.check_numbers_prime_concurrently()
        count = sum(1 for result in results if result[1])
        delta_time = time.time() - t0
        assert count == 112
        assert delta_time < 5

File: single-language-analysis/dataset-files/java/TestSmell133.java
import org.junit.Test;
import org.junit.Ignore;
import static org.junit.Assert.*;

public class TestSmell133 {

    @Ignore("5 seconds is also too long")
    @Test
    public void testPrimesParallel() {
        long t0 = System.currentTimeMillis();
        boolean[][] results = checkNumbersPrimeConcurrently();
        int count = 0;
        for (boolean[] result : results) {
            if (result[1]) {
                count++;
            }
        }
        double deltaTime = (System.currentTimeMillis() - t0) / 1000.0;
        assertEquals(112, count);
        assertTrue(deltaTime < 5);
    }
}